docs/serving/using-cert-manager-on-gcp.md

```diff
@@ -1,241 +0,0 @@
----
```

Comment:
Make sure this wasn't referenced anywhere?

Reply:
Done